test/emails/spam.txt

```diff
@@ -0,0 +1,75 @@
+> On Thu, Apr 30, 2020 at 10:07 AM, "someone" <someone@example.com> wrote:
```

Comment:
Should there also be some actual content to ensure we properly trim/elide?

Reply:
I've added the test. :blush: